buildspec.json

```diff
@@ -1,29 +1,29 @@
 {
     "dependencies": {
         "prebuilt": {
-            "version": "2023-04-03",
-            "baseUrl": "https://github.com/obsproject/obs-deps/releases/download",
+            "version": "2023-04-07",
+            "baseUrl": "https://github.com/DDRBoxman/obs-deps/releases/download",
             "label": "Pre-Built obs-deps",
             "hashes": {
-                "macos-x86_64": "2dfa6f7d14e576ab62abbaba60725f328e4fdcb30a07f1ac0a9d65f00825822d",
-                "macos-arm64": "761904548b31f1a06b6a2afb745fde71655d5d86802dc3b64917151c95c30f16",
-                "macos-universal": "0939339b45ad426d5649e7db945f1c479fde075418efc7789489758e7ecbc115",
-                "windows-x64": "01b863b76b8ec55d870986d0649fe5b16eff1303179b4b2465e2eb031f3e4abf",
-                "linux-x86_64": "3368b74195c31214051e92bac1ba4f99fb649d4d755e4611e9d7fc85c8375862"
+                "macos-x86_64": "afb1eba964e24c8e82e5f57987e72d192d7aaff8b0ee7b7df9f4b78a1483d70d",
+                "macos-arm64": "9de411c5bd12b2e7305178d8fbbe29c8a243e561896082b82e330439129ad03a",
+                "macos-universal": "c73c7d85a02257206f7acea9f1068a06155874658e03492ecebeba37936d4a76",
+                "windows-x64": "e4f6bb9073f11d9c5e3cb089a824c0b7d9bd2c697617641cfb4a62d8c7a3c247",
+                "linux-x86_64": "9af164dbd007d4fc2b0cbc00f8bed2d45422c815d5a79dcb27aff8708fb1fe33"
             }
         },
         "qt6": {
-            "version": "2023-04-03",
+            "version": "2023-04-07",
```

Comment:
Probably don't need to point to the new qt deps

Reply:
Temp commit anyway, we can ignore